### PR TITLE
feat(shorebird_cli): expose user

### PR DESCRIPTION
**Pull request by felangel**
## Description

- feat(shorebird_cli): expose user

## Type of Change

- [X] ✨ New feature (non-breaking change which adds functionality)
- [ ] 🛠️ Bug fix (non-breaking change which fixes an issue)
- [ ] ❌ Breaking change (fix or feature that would cause existing functionality to change)
- [ ] 🧹 Code refactor
- [ ] ✅ Build configuration change
- [ ] 📝 Documentation
- [ ] 🗑️ Chore

### CODE REVIEW

File: packages/shorebird_cli/lib/src/auth/auth.dart [hunk 1]
```diff
@@ -75,7 +78,11 @@ class Auth {
 
   AccessCredentials? _credentials;
 
-  AccessCredentials? get credentials => _credentials;
```
Comment (felangel) [inline packages/shorebird_cli/lib/src/auth/auth.dart:78]: The `user` is exposed instead of `credentials`

**Review by eseidel — APPROVED** [review]

File: packages/shorebird_cli/lib/src/commands/login_command.dart [hunk 2]
```diff
@@ -30,7 +29,7 @@ class LoginCommand extends ShorebirdCommand {
       await auth.login(prompt);
       logger.info('''
 
-🎉 ${lightGreen.wrap('Welcome to Shorebird! You are now logged in.')}
+🎉 ${lightGreen.wrap('Welcome to Shorebird! You are now logged in as <${auth.user!.email}>.')}
```
Comment (bryanoltman) [inline packages/shorebird_cli/lib/src/commands/login_command.dart:32]: Nice! Making a TODO to add this to `doctor`.

File: packages/shorebird_cli/lib/src/auth/auth.dart [hunk 3]
```diff
@@ -109,3 +118,17 @@ class Auth {
     _httpClient.close();
   }
 }
+
+extension on AccessCredentials {
+  User toUser() {
+    final claims = Jwt.decodeClaims(idToken ?? '');
```
Comment (bryanoltman) [inline packages/shorebird_cli/lib/src/auth/auth.dart]: Can we throw if `idToken` is null? It's an entirely different issue than an invalid token, even if we ultimately handle it the same way.

**Review by bryanoltman — APPROVED** [review]
Love it

File: packages/shorebird_cli/lib/src/auth/auth.dart (hunk 3 above)
Comment (felangel) [reply packages/shorebird_cli/lib/src/auth/auth.dart]: Yup good call, updated ✅

**Pull request merged**